rawfile/rawfile.py

```diff
@@ -34,15 +38,36 @@ def csi_driver(driver_config: CSIDriverCmd):
     server = grpc.server(
         futures.ThreadPoolExecutor(max_workers=int(driver_config.grpc_workers))
     )
+    bg_task_executor = ThreadPoolExecutor(max_workers=5)
+    _task_manager = task_manager.TaskManager(bg_task_executor)
     internal_server = None
     csi_pb2_grpc.add_IdentityServicer_to_server(
-        bd2fs.Bd2FsIdentityServicer(rawfile_servicer.RawFileIdentityServicer()), server
+        bd2fs.Bd2FsIdentityServicer(
+            rawfile_servicer.RawFileIdentityServicer(),
+            task_manager=_task_manager,
+        ),
+        server,
     )
     if driver_config.plugin_type == "node":
         migrate_all_volume_schemas()
+        cow_test_src = Path(f"{consts.DATA_DIR}/cow.test.src")
```

Comment:
maybe move this logic to the filesystem/init.py ?

Reply:
I will add some startup hooks, I will move some parts there